keps/sig-node/375-cpu-manager/README.md

```diff
@@ -95,7 +143,34 @@ to build and test new policies. The shared state abstraction allows
 other Kubelet components to be agnostic of the CPU manager policy for
 observability and checkpointing extensions._
 
-#### Discovering CPU topology
+### User Stories (Optional)
+
+#### Story 1 : High-performance applications
+
+Systems such as real-time trading system or 5G CNFs (User Plain Function, UPF) need to maximize the CPU time; CPU pinning ensure exclusive CPU allocation and allows to avoid performance issues due to core switches, cold caches.
```

Comment:
Isn't it userplane function?

```suggestion
Systems such as real-time trading system or 5G CNFs (User Plane Function, UPF) need to maximize the CPU time; CPU pinning ensure exclusive CPU allocation and allows to avoid performance issues due to core switches, cold caches.
```

Reply:
it is, it's a typo. I'll fix in the next upload, thanks.